src/sphinx.md

```diff
@@ -242,6 +249,9 @@ route](./topology.md#route-generation):
 - One forward-to-peer-ID action (50 bytes).
 - One deliver action with a 16-byte ID (18 bytes).
 
+// TODO this note on the different actions size and worst_case_route and how they can compose is very important, could
+// be in its own chapter (not much info in the linked chapter).
```

Comment:
It's important, but I'm not sure there's much more to say than this?

Reply:
yes that is a bit awkward, maybe the usual frame schema with numbers, but at the end the important is that the information is visible, so just having a specific chapter/header might do the job.

Reply:
Is it not visible enough here? Ultimately nodes don't really need to care about this: if they just follow the rules for route generation, the actions will always fit.

Reply:
as long as it is part of the spec it is good, just don't want it to be diluted in less relevant info. Just mentioning (since it differs from usual sphinx info it is something that would be good to be hard to miss).